gio-subclass/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+extern crate libc;
+#[macro_use]
+extern crate glib;
+extern crate gio;
+
+extern crate glib_sys as glib_ffi;
+extern crate gobject_sys as gobject_ffi;
+extern crate gio_sys as gio_ffi;
+
+#[macro_use]
+extern crate gobject_subclass;
+
+// If we make application private (which it should be), `box_gapplication_impl` stops working.
+pub mod application;
+pub use application::*;
```

Comment:
Elsewhere we don't re-export the whole module content via the crate but only the module itself. I'd prefer to keep this opt-in for users of the crates (and this would also solve your comment above)

Reply:
I was trying to mirror what they did in the main gtk-rs repo's. But indeed, it would solve those problems if we didn't re-export the contents